skimage/feature/corner.py

```diff
@@ -917,16 +918,34 @@ def corner_subpix(image, corners, window_size=11, alpha=0.99):
 
 def corner_peaks(image, min_distance=1, threshold_abs=None, threshold_rel=None,
                  exclude_border=True, indices=True, num_peaks=np.inf,
-                 footprint=None, labels=None, *, num_peaks_per_label=np.inf):
+                 footprint=None, labels=None, *, num_peaks_per_label=np.inf,
+                 p_norm=np.inf):
     """Find corners in corner measure response image.
```

Comment:
I know you haven't touched this, but I see now that it's a confusing phrasing.

How about: `Find peaks in corner response image, corresponding to corners in the original image.`

Reply:
Isn't `Find peaks in a corner response image.` clear enough?